packages/server/src/automations/tests/utilities/index.ts

```diff
@@ -20,6 +21,17 @@ export function afterAll() {
   config.end()
 }
 
+export function getTestQueue(): queue.InMemoryQueue<AutomationData> {
+  return getQueue() as unknown as queue.InMemoryQueue<AutomationData>
+}
```

Comment:
This is what I was trying to avoid in the previous PR, but I bit the bullet and did it. I wanted to just use methods on the `Bull.Queue` interface but that was holding me back from being able to manually retrigger cron messages.